README.md

```diff
@@ -34,33 +34,28 @@ export base64_capture_macs_script_content=$(cat capture-macs.sh|base64 -w 0) &&
 fcct custom-config.fcc > file.ign
 ```
 
-  - Upload `file.ign` to a shared location which the OpenShift nodes can access.
+  - Upload `file.ign` to a shared location where the OpenShift nodes can access.
 
-- Base64 encode the `setup-ovs.sh` file and paste the content into each MCO file into "base64_script_content" section. 
+- Base64 encode the `init-interfaces.sh` file and paste the content into each MCO file into "base64_script_content" section. 
 
     **TIP:** To update the content you can use:
 
 ```
-export base64_script_content=$(cat setup-ovs.sh|base64 -w 0) && envsubst <  mco_ovs_workers.yml.tmpl > mco_ovs_workers.yml && envsubst < mco_ovs_supervisor.yml.tmpl > mco_ovs_supervisor.yml
+export base64_script_content=$(cat init-interfaces.sh|base64 -w 0) && envsubst <  mco_ovs_workers.yml.tmpl > mco_ovs_workers.yml && envsubst < mco_ovs_supervisor.yml.tmpl > mco_ovs_supervisor.yml
 ```
 
-- MCO files are MachineConfig, which you can apply manually once the cluster is up or add it to the installation automation/pipeline.
```

Comment:
Way remove? Some information about MCO(and what does it mean)

Reply:
Didn't remove it, just moved it to a more relevant place [here](https://github.com/RHsyseng/rhcos-slb/pull/44/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52)